### ISSUE TITLE
Missing file

### ISSUE THREAD

**Issue by ArashRabbani** [stke9/SliceGAN#13]
Hi there, 
In networks.py we have:

    if Training:
        with open(pth + '_params.data', 'wb') as filehandle:
            # store the data as binary data stream
            pickle.dump(params, filehandle)
    # if loading model, load the associated params file
    else:
        with open(pth + '_params.data', 'rb') as filehandle:
            # read the data as binary data stream
            dk, ds, df, dp, gk, gs, gf, gp  = pickle.load(filehandle)

However, the file named '*_params.data'  is not included in the repository. Can you please provide some guidance? 

Thanks a lot

**Comment by stke9** [comment]
Hi Arash,

A params file is created automatically each time you run slicegan in train mode (terminal command: python run_slicegan.py 1). Params contains info about the network size and shape, to allow you to load networks correctly when you run slicegan in generate mode after training is complete.

**Comment by ArashRabbani** [comment]
Thanks a lot for your response, 
I was wondering if there is a pre-trained model available for experimentation. 
Thanks for your help

**Comment by stke9** [comment]
There are 77 different pretrained generators available on the [microlib.io](https://microlib.io/) website.You can see how to download and run them on the microlib github in the last section of the [ github](https://github.com/tldr-group/microlib) README:

**Comment by ArashRabbani** [comment]
Thanks a lot, Steve. The website was amazing, I can't believe I didn't know about this strong work of yours. Thanks a lot for sharing.

**Comment by stke9** [comment]
Thanks Arash, glad you thought so! let me know if you manage to download the generators and get them working.

**Comment by ArashRabbani** [comment]
Sure, thank you very much. I was wondering if any experimental data are available for the geometries on microlib website. That could give the opportunity to make physical models that predict the experimental properties.

**Comment by ArashRabbani** [comment]
Thanks for your help. The data source provides more information. It is very useful.
